CODEOWNERS

```diff
@@ -1 +0,0 @@
-* @lcatherine @lyze
```

Comment:
This file is deleted now, which is *not* what we want. Please restore this file.

Reply:
done